Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -546,12 +550,14 @@ inline void SeedFinder<external_spacepoint_t, platform_t>::filterCandidates(
         // x_0 and y_0
         float A0 = (lt.V - Vb) / dU;
 
+        float zPositionMiddle = cosTheta * std::sqrt(1 + A0 * A0);
```

Comment:
```suggestion
        float zPositionMiddle = cosTheta * std::hypot(1, A0);
```

Reply:
Thank you @andiwand for the suggestions. Isn't `std::hypot` a lot slower than `std::sqrt(x*x + y*y)`?

Reply:
right - I guess it depends on your accuracy vs performance targets. feel free to resolve my comments